index.js

```diff
@@ -0,0 +1,22 @@
+const hamburger = document.getElementById('mobile-nav');
+const navUl = document.getElementById('mobile-nav-tog');
+const cancelBut = document.getElementById('cancel-butt');
+const cancelBut1 = document.getElementById('cancel1-butt');
+const cancelBut2 = document.getElementById('cancel2-butt');
+const cancelBut3 = document.getElementById('cancel3-butt');
```

Comment:
These are not actually buttons but the links to other sections like `projects`, `contact-us` and `about`. instead of giving these this `id` that doesn't describe them. why don't we just have one class name to all of them and query the class name instead?

Say we use a class name of `nav-deactivator` our code can be like

```js
const navDeactivators = document.querySelectorAll('.nav-deactivator');

const removeActiveClass = () => {
  navUl.classList.remove('active');
}

navDeactivators.forEach(navDeactivator => {
  navDeactivator.addEventListener('click', removeActiveClass);
});
```
By doing this, if we create more links in the mobile nav in the future, just adding the `nav-deactivator` class will give the link the functionality to deactivate the mobile nav